cognee/tasks/graph_decomposition_to_graph_nodes/graph_decomposition_to_graph_nodes.py

```diff
@@ -0,0 +1,219 @@
+import json
+import asyncio
+from uuid import uuid5, NAMESPACE_OID
+from datetime import datetime, timezone
+from typing import Type
+from pydantic import BaseModel
+from cognee.infrastructure.databases.graph import get_graph_engine
+from cognee.infrastructure.databases.vector import DataPoint, get_vector_engine
+from cognee.modules.data.extraction.knowledge_graph.extract_content_graph import extract_content_graph
+from cognee.modules.data.processing.chunk_types.DocumentChunk import DocumentChunk
+
+
+class EntityNode(BaseModel):
+    uuid: str
+    name: str
+    type: str
+    description: str
+    created_at: datetime
+    updated_at: datetime
+
+async def graph_decomposition_to_graph_nodes_task(data_chunks: list[DocumentChunk], graph_model: Type[BaseModel], collection_name: str):
```

Comment:
`graph_decomposition_to_graph_nodes_task` -> `decompose_chunks_into_graph` or something like that.